src/hooks/useActiveProject.tsx

```diff
@@ -63,10 +63,14 @@ export const ActiveProjectContextProvider = ({
   children?: React.ReactNode;
 }) => {
   const projectsResult = useSubjectProjects();
+  const projectLoading = projectsResult.isLoading || !projectsResult.isFetched;
```

Comment:
I wonder if all we need is `!projectsResult.isFetched`. If the data is fetched but the hook still says it is loading does it matter? I would think we would only really need to monitor `isLoading` if we are doing a refetch or loading more results.